convert2rhel/checks.py

```diff
@@ -284,13 +284,20 @@ def ensure_compatibility_of_kmods():
                 unsupported_kmods,
             )
         )
-        logger.critical(
-            (
+        if "CONVERT2RHEL_UNSUPPORTED_UNCHECKED_KMODS" in os.environ:
+            logger.warning(
+                "The following kernel modules are not supported in RHEL:\n{kmods}\n"
+                "'CONVERT2RHEL_UNSUPPORTED_UNCHECKED_KMODS' environment variable detected, continuing conversion."
+            ).format(kmods=not_supported_kmods, system=system_info.name)
+
+        else:
+            logger.critical(
                 "The following kernel modules are not supported in RHEL:\n{kmods}\n"
                 "Make sure you have updated the kernel to the latest available version and rebooted the system. "
-                "Remove the unsupported modules and run convert2rhel again to continue with the conversion."
+                "Only kernel modules supported in RHEL are preferd for conversion, if you want to ignor this "
+                " check, set the environment variable 'CONVERT2RHEL_UNSUPPORTED_UNCHECKED_KMODS' to continue"
             ).format(kmods=not_supported_kmods, system=system_info.name)
```

Comment:
```suggestion
logger.critical(
                "The following kernel modules are not supported in RHEL:\n{kmods}\n"
                "Make sure you have updated the kernel to the latest available version and rebooted the system. "
                "Only kernel modules supported in RHEL are preferd for conversion, if you want to ignor this "
                " check, set the environment variable 'CONVERT2RHEL_UNSUPPORTED_UNCHECKED_KMODS' to continue".format(kmods=not_supported_kmods, system=system_info.name)
            )
```

The same here is true as the other comment I made about where to place the `.format()` function call.

Reply:
gotcha! thank you again for the help with all of your comments for this PR!